pkg/cmd/pulumi/packagecmd/project_sdk_link.go

```diff
@@ -733,9 +733,6 @@ func SchemaFromSchemaSource(pctx *plugin.Context, packageSource string, args []s
 	if pluginSpec.PluginDownloadURL != "" {
 		spec.PluginDownloadURL = pluginSpec.PluginDownloadURL
 	}
-	if pluginSpec.Version != nil {
```

Comment:
I think it's fine to leave this here

Reply:
I'm actually wondering if we should pull all of this into `GetSchema`, so we always get a consistent schema, and don't just have things for generating the SDK :thinking: 

But we can leave it here for now.